external/llvm-project/mlir/test/Conversion/SoftwareBF16/softwareBF16.mlir

```diff
@@ -0,0 +1,57 @@
+// RUN: mlir-opt --llvm-software-bf16 %s| FileCheck %s
+
+module attributes {llvm.data_layout = ""} {
+  llvm.func @verify_bf16_f32(%arg0: bf16, %arg1: f32) -> i32 {
+//CHECK:  llvm.func @verify_bf16_f32(%arg0: i16, %arg1: f32) -> i32 {
+
+    %0 = llvm.mlir.constant(0 : i32) : i32
+    %1 = llvm.mlir.constant(1 : i32) : i32
+//CHECK:    %[[V0:.*]] = llvm.mlir.constant(32767 : i32) : i32
```

Comment:
CHECK-DAG on the constants. Also, I'm not convinced by the extra whitespace here